### PR TITLE
Support for big definition overrides

### PR DESCRIPTION
**Pull request by BrunoBernardino**
Using an RPC call to get those.

Also adds support for building a boilerplate zip with the current "local" changes, vs the latest npm package.

This needs https://github.com/zapier/zapier/pull/17392 to ship before merging this PR.

Related to [PDE-166](https://zapierorg.atlassian.net/browse/PDE-166).

### CODE REVIEW

**Review by eliangcs — APPROVED** [review]
Tested with the backend. It works! 👍

**Comment by BrunoBernardino** [comment]
@eliangcs would you mind checking again? I've got definition caching working now!

File: integration-test/integration-test.js [hunk 1]
```diff
@@ -56,7 +57,7 @@ runLocally.testName = 'runLocally';
 
 const doTest = runner => {
   describe(`${runner.testName} integration tests`, () => {
-    it('should return data from app function call', done => {
+    it('should return data from app function call', () => {
```
Comment (eliangcs) [inline integration-test/integration-test.js:60]: Nice style change!

**Review by eliangcs — APPROVED** [review]
The change looks good! I assume you already tested it so I didn't pull.

**Pull request merged**